depthai_nodes/ml/messages/creators/classification_sequence.py

```diff
@@ -0,0 +1,88 @@
+from typing import List, Union
+
+import numpy as np
+
+from .. import Classifications
+
+
+def create_classification_sequence_message(
+    classes: List,
+    scores: Union[np.ndarray, List],
+    remove_duplicates: bool = False,
+    ignored_indexes: List[int] = None,
+) -> Classifications:
+    """Creates a message for a multi-class sequence. The 'scores' array is a sequence of
+    probabilities for each class at each position in the sequence. The message contains
+    the class names and their respective scores, ordered according to the sequence.
+
+    @param classes: A list of class names, with length 'n_classes'.
+    @type classes: List
+    @param scores: A numpy array of shape (sequence_length, n_classes) containing the (row-wise) probability distributions over the classes.
+    @type scores: np.ndarray
+    @param remove_duplicates: If True, removes consecutive duplicates from the sequence.
```

Comment:
Maybe explain this duplicate removal better as it not really clear what is removed and why